osbuild.py

```diff
@@ -196,6 +193,60 @@ def print_header(title, options, machine_name):
     print(f"\t# nsenter -a --wd=/root -t `machinectl show {machine_name} -p Leader --value`")
     print()
 
+class Stage:
+    def __init__(self, stage, objects):
```

Comment:
Initializing a `Stage` object with a `stage` dict is awkward. Can we explode the arguments? (Same for `Assembler`)

Reply:
Right, agreed. Will do.